functests/5_latency_testing/latency_testing.go

```diff
@@ -102,6 +105,24 @@ var _ = table.DescribeTable("Test latency measurement tools tests", func(testGro
 				testlog.Error(err.Error())
 			}
 			Expect(string(output)).NotTo(MatchRegexp(unexpectedError), "Unexpected error was detected in a positve test")
+			//Check runtime argument in the pod's log only if the tool is expecetd to be executed
+			res, err := regexp.MatchString(success, string(output))
```

Comment:
in golang, we usually name the bool return value like `ok` or `matched` for regexps. Next time, it'll look a bit better like
```
matched, err := regexp.MatchString(success, string(output))
if err != nil {
  testlog.Error(err.Error())
}
if matched {
   /...
}
```

Reply:
sounds better indeed, I will consider that next time, thank you!